operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -841,10 +857,15 @@ func (rc *ReconciliationContext) UpdateCassandraNodeStatus() error {
 			if nodeStatus.HostID == "" {
 				endpointsResponse, err := rc.NodeMgmtClient.CallMetadataEndpointsEndpoint(pod)
 				if err == nil {
+					useHostIP := rc.rpcAddressIsHostIP()
```

Comment:
Would we want to make a GetRpcAddress() that returns the HostIP or PodIP, whichever is configured?

Reply:
great idea!